sdk/examples/gateway.ts

```diff
@@ -51,13 +49,8 @@ async function createTxWithOpReturn(fromAddress: string, toAddress: string, amou
     );
 
     const psbt = unsignedTx.toPSBT(0);
-    const psbtHex = hex.encode(psbt);
 
-    // TODO: sign PSBT
```

Comment:
Technically this comment is still valid, the code doesn't sign anything in this example